DSCResources/MSFT_xSQLServerSetup/MSFT_xSQLServerSetup.psm1

```diff
@@ -874,7 +928,11 @@ function Test-TargetResource
         $ASConfigDir,
 
         [System.Management.Automation.PSCredential]
-        $ISSvcAccount
+        $ISSvcAccount,
+
+        [System.String]
+        [ValidateSet("Automatic", "Disabled", "Manual")]
+        $BrowserSvcStartupType
```

Comment:
By default Browser service is in disabled mode after installation.
If you are installing named instance and want to use dynamic ports Browser service needs to be running.

I don't think we need any default value in Resource level, if this is omitted then it is not passed in to setup.exe so behaviour stays same. So I think this change is backwards compatible and is safe to add.